torch/_inductor/codecache.py

```diff
@@ -102,9 +102,8 @@ def cpp_compiler_search(search):
     for cxx in search:
         try:
             if cxx is None:
-                # gxx package is only available for Linux
-                # according to https://anaconda.org/conda-forge/gxx/
-                if sys.platform != "linux":
+                # Do not attempt to install gcc-12 by default
+                if os.getenv("TORCH_INDUCTOR_INSTALL_GCC_VIA_CONDA"):
```

Comment:
Can we add a note in the config.py about the need of setting this new env to get conda installation work?
https://github.com/pytorch/pytorch/blob/7b0650d5cf4897089f32c011504d2b2d185cc60a/torch/_inductor/config.py#L105

Reply:
Done